src/main/java/gov/cabinetoffice/gap/adminbackend/entities/GrantMandatoryQuestions.java

```diff
@@ -84,17 +86,10 @@ public class GrantMandatoryQuestions extends BaseEntity {
     @Enumerated(EnumType.STRING)
     @ColumnTransformer(write = "?::grant_mandatory_question_status")
     @Builder.Default
-    private GrantMandatoryQuestionStatus status = GrantMandatoryQuestionStatus.NOT_STARTED; // TODO
-                                                                                            // what
-                                                                                            // if
-                                                                                            // the
-                                                                                            // status
-                                                                                            // is
-                                                                                            // still
-                                                                                            // in
-                                                                                            // progress
-
-    @Column(name = "version", nullable = false)
+    private GrantMandatoryQuestionStatus status = GrantMandatoryQuestionStatus.NOT_STARTED;
+
+    // TODO what if the status is still in progress
```

Comment:
Not sure what this comment is asking? 

The `Builder.Default` method only runs on construction of a new instance of the object so if the status is still in progress then it won't change. If we need to create a new instance of the object with a status of in progress then we can do it through the builder or we can use the `setStatus` method after creation.